pandas/tests/formats/test_format.py

```diff
@@ -3938,6 +3938,15 @@ def test_period(self):
         self.assertEqual(str(df), exp)
 
 
```

Comment:
in fact fixture setup / changing to parameterized is actually quite a bit of work :> we have *so* many loop-y type tests which really should be parametrized.